libs/core/langchain_core/utils/tool_examples.py

```diff
@@ -0,0 +1,111 @@
+import uuid
```

Comment:
Look good. Do we think it's best to create tool_examples or put into function_calling module?

Reply:
ya moved